### DISCUSSION TITLE
Looks like "Discussion Fee"

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#2811] [Discussions] [answered]
This looks like there is a fee for making a discussion.
![18027-0](https://user-images.githubusercontent.com/1230959/100397382-54b69400-3084-11eb-8359-a82c335acfa0.jpg)
Perhaps don't truncate the word so fast.
There certainly is plenty of whitespace remaining.

**Comment by becca** [comment, marked as answer]
Oh no! There most certainly is no fee for creating a discussion here 😄 

Thank you for letting me know - we'll look into fixing this and report back. 🙇
